airscan.go

```diff
@@ -314,6 +333,31 @@ func (c *Client) Scan(settings *ScanSettings) (*ScanState, error) {
 		}
 	}
 
+	// Check capabilities
+	caps, err := c.ScannerCapabilities()
+	if err != nil {
+		return nil, err
+	}
+
+	if settings.InputSource == "Feeder" {
+		if caps.Adf == nil {
+			return nil, fmt.Errorf("this scanner doesn't have an ADF")
+		}
+
+		if settings.Duplex && caps.Adf.AdfDuplexInputCaps == nil {
+			return nil, fmt.Errorf("this scanner doesn't support duplex mode")
+		}
+
+		if !settings.Duplex && caps.Adf.AdfSimplexInputCaps == nil {
```

Comment:
I think it might be prudent to remove this check. Simplex ought to work, and I’m not sure whether some devices might not report their capabilities correctly.

Reply:
Can you remove the simplex check?